### ISSUE TITLE
Sending a message to a website user - Requires name/email/phone doesn't function

### ISSUE THREAD

**Issue by groovybluedog** [LiveHelperChat/livehelperchat#1629]
It's possible I'm misunderstanding this functionality, but when sending a message to the user, no matter what combination of "requires name", "requires email" or "requires phone" is selected, when the end user clicks the bubble/prompt to respond, the chat opens and they can reply with a message immediately. At no point are they asked for the items selected when sending the message.

![image](https://user-images.githubusercontent.com/16142383/110597964-206d6c00-8179-11eb-9683-e03a7d11236d.png)

![image](https://user-images.githubusercontent.com/16142383/110598070-4135c180-8179-11eb-8d64-4e3906b94e03.png)

![image](https://user-images.githubusercontent.com/16142383/110598103-48f56600-8179-11eb-9f22-0f0f9a27e157.png)

**Comment by remdex** [comment]
I don't think this workflow is supported anymore in general only in old one widget it was. If you want to ask visitor username/phone etc. You can configure start chat form settings and have them always requested.
